envoy/stats/allocator.h

```diff
@@ -70,11 +70,11 @@ class Allocator {
   virtual void markTextReadoutForDeletion(const TextReadoutSharedPtr& text_readout) PURE;
 
   /**
-   * Iterate over all stats that need to be sinked. Note, that implementations can potentially hold
+   * Iterate over all stats that need to be sunk. Note, that implementations can potentially hold
```

Comment:
im not totally sure if this is correct to change, as `sink` in this usage is terminology - and is an expansion of the noun sink - it kinda loses its meaning changing it to sunk

perhaps we could change the text, so its proper engish *and* doesnt change the usage of sink

something like

> Iterate over all the stats that need to be added to a sink

Reply:
apart from anything else it would disambiguate from the stats being `synced`